doc/reference/simple-scripts.md

```diff
@@ -271,3 +271,96 @@ cardano-cli shelley transaction assemble \
   --out-file spendMultiSig
 ```
 You can now submit this tx via `cardano-cli shelley transaction submit`!
+
+
+# Simple Script
+
+In the Allegra era, we introduce the ability to time lock your multisignature scripts. From now on we will refer to these scripts (collectively) as simple scripts and differentiate them based on the era they are present in and which features are available. That is, in the Shelley era we do not have time locking available in our scripts but in the Allegra era we do.
+
+NB: If you specify `"type": "any"` this will allow you to spend from your script address regardless of what time lock you have declared.
+
+## Bounds
+
+It is important to understand the bounds on a transaction and how they relate to the time locking scripts. Previously we had a `ttl` argument in the cli which was an upper bound on a transaction i.e the transaction was valid up until that slot number. This has been replaced with `--lower-bound` and `--upper-bound` arguments as described below:
+
+```
+If you specify a lower bound on a transaction it is valid from the lower-bound:
+  lower bound           +infinity
+       |------------------->
+If you specify an upper bound on a transaction it is valid to upperbound - 1:
+  -infinity           upper bound - 1
+       <-------------------|
+If you specify both bounds the tx is valid from the lower bound to the upperbound - 1
+  lower bound         upper bound - 1
+       <------------------->
+```
```

Comment:
These diagrams are useful, but you could also use interval notation here since you can specify whether an endpoint is inclusive or exclusive. Example: `[lowerBound, infinity)`, `(-infinity, upperBound)`, `[lowerBound, upperBound)`